sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -545,11 +545,11 @@ def from_config(
 
         :param credential: The credential object for the workspace.
         :type credential: ~azure.core.credentials.TokenCredential
-        :param path: The path to the config file or starting directory to search.
+        :keyword path: The path to the config file or starting directory to search.
```

Comment:
So all keyword args need to be keyword instead of param?

Reply:
Yes. The azure-pylint-guidelines-checker will produce the `docstring-should-be-keyword` message for parameters that are keyword only.